tests/test_modern.py

```diff
@@ -493,6 +493,70 @@ def test_abort_confirm(mock_input):
         pyclean.modern.confirm('Abort execution')
 
 
+@patch('pyclean.modern.print_dirname')
+@patch('pyclean.modern.print_filename')
+@patch('builtins.input')
+@patch(
+    'pathlib.Path.glob',
+    return_value=[
+        DirectoryMock(),
+        FileMock(),
+    ],
+)
```

Comment:
I think, this fits all in one line and neither the linter nor the formatter will complain.

```suggestion
@patch('pathlib.Path.glob', return_value=[DirectoryMock(), FileMock()])
```